### PR TITLE
Merge changes from TAU 1.2-base into TAU 1.2

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

**Comment by singa2000** [comment]
- Please remove the  ./mobile/theme-changeable/images/smartthings/ folder.
- src/css/profile/mobile/common folder is created for tau_1.2 ? Is it different with src/css/profile/mobile/changeable folder??

**Comment by TomaszLukawskiSam** [comment]
>     * Please remove the  ./mobile/theme-changeable/images/smartthings/ folder.
Thank you. I deleted this folder once. Something went wrong with merge.

> 
>     * src/css/profile/mobile/common folder is created for tau_1.2 ? Is it different with src/css/profile/mobile/changeable folder??
It doesn't matter much. In the new version, graphic topics are not changeable. We are using css variables.

**Comment by lslachciak** [comment]
shouldn't this PR be called - "merge changes from TAU 1.2-base into..." ?

**Comment by TomaszLukawskiSam** [comment]
Several issues refer to this PR (#619 #620 #621)

File: tests/js/core/widget/core/Page/Page.js [hunk 1]
```diff
@@ -598,6 +598,8 @@
 		});
 
 		test("_setTitle", function () {
+			ok(true, "Test disabled");
```
Comment (lslachciak) [inline tests/js/core/widget/core/Page/Page.js:601]: Is this Qunit test right?
shouldn't we skip method instead?
skip("_setTitle", function () {...{

https://api.qunitjs.com/QUnit/skip

**Comment by lslachciak** [comment]
please also remove references to smartthings from
./src/css/profile/mobile/common/gridview.less
./src/css/profile/mobile/common/radio.less
./src/css/profile/mobile/common/dropdownmenu.less
./src/css/profile/mobile/common/button.less
./src/css/profile/mobile/common/checkbox.less

File: tests/js/core/widget/core/Page/Page.js (hunk 1 above)
Comment (TomaszLukawskiSam) [reply tests/js/core/widget/core/Page/Page.js:601]: The Quinit version used in the project does not have the "skip" command

**Comment by TomaszLukawskiSam** [comment]
> please also remove references to smartthings from
> ./src/css/profile/mobile/common/gridview.less
> ./src/css/profile/mobile/common/radio.less
> ./src/css/profile/mobile/common/dropdownmenu.less
> ./src/css/profile/mobile/common/button.less
> ./src/css/profile/mobile/common/checkbox.less

This comment refers to old PR. Current PR has modified these files.

**Review by lslachciak — APPROVED** [review]

**Pull request merged**